client/verta/verta/_internal_utils/model_dependencies.py

```diff
@@ -4,7 +4,7 @@
 from types import ModuleType
 from typing import Callable, get_type_hints, Set, Type
 
-from verta.registry import VertaModelBase
+from ..registry._verta_model_base import VertaModelBase
```

Comment:
Out of curiosity, was there a particular reason for this change?

```python
from verta.registry import VertaModelBase
```

would've been the more canonical way to do this import in my opinion, though we don't really have a strict guideline for it.